JM/CF/Scripts/3_Game/CommunityFramework/RPC/RPCManager.c

```diff
@@ -185,6 +185,24 @@ class RPCManager
 
 		return true;
 	}
+
+	bool RemoveRPC( string modName, string funcName )
+	{
+		#ifdef CF_TRACE_ENABLED
+		auto trace = CF_Trace_2(this).Add(modName).Add(funcName);
+		#endif
+
+		if ( m_RPCActions.Contains( modName ) )
+		{
+			if ( m_RPCActions[ modName ].Contains( funcName ) )
+			{
+				m_RPCActions[ modName ].Remove( funcName );
+				return true;
+			}
```

Comment:
If you remove something you do not need to check if it existed. Remove should probably also return a bool if was successful or not.

Afterwards a check for `m_RPCActions[modName].Count() == 0` to also do `m_RPCActions.Remove(modName)` might make sense to clean it up.

Reply:
Default map.Return method doesn't return an execution status, that's why it is wrapped in if operator. I'll change it from bool to void